src/components/EmptyArea.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Center, VStack, Text, HTMLChakraProps } from "@chakra-ui/react"
+import { ReactFragment } from "react"
+
+import { EmptyBoxImage } from "assets"
+
+export interface EmptyAreaOption extends HTMLChakraProps<"div"> {
+  isItemEmpty: boolean
+  actionButton: ReactFragment
```

Comment:
this might be wrong - `ReactFragment` specifically refers to `<> </>` ._.  as an aside, this actually accepts jsx elements, which i didn't know so til! 

the side effect of this is that now, this becomes a required prop, which isn't ideal - wdyt about chagning this to `JSX.Element`?